core/templates/components/state-editor/state-editor-properties-services/state-hints.service.ts

```diff
@@ -35,6 +35,8 @@ export class StateHintsService extends StatePropertyService<Hint[]> {
     this.setterMethodKey = 'saveHints';
   }
 
+  displayed: Hint[] =[];
+
```

Comment:
Why is this required?

Reply:
`state-hint-editor.component.html` is using `stateHintsService.displayed.length` and if we don`t use initializes displayed with `[ ] empty array` then in e2e test there was an error `cannot read undefined reading length` so to fix the and make displayed define I used to pass empty array in it.

Reply:
Sg! Just a nit:

```suggestion
  displayed: Hint[] = [];
```

Reply:
Done